account_invoice_currency/README.rst

```diff
@@ -7,13 +12,39 @@ amount untaxed, amount taxed and amount total.
 These fields are shown in "Other information" tab in invoice form.
 
 
+Usage
+=====
+
+**NOTE #1**: Amount untaxed and amount total are official account fields in v9.
+This addon adds amount taxes and shows all of them in invoice form.
+
+**NOTE #2**: Please note that these fields are negative (as done in v9 account addon)
+for credit notes (for example a refund invoice)
+
+
+Known issues / Roadmap
+======================
+
+These fields name change from v8:
+
+* cc_amount_untaxed -> amount_untaxed_signed (already in account v9)
```

Comment:
Please prepare a migration script (using openupgradelib) that renames the columns and invert sign of the corresponding invoices.